tico/quantization/wrapq/wrappers/nn/quant_silu.py

```diff
@@ -19,13 +19,13 @@
 
 from tico.quantization.config.ptq import PTQConfig
 from tico.quantization.wrapq.wrappers.quant_module_base import QuantModuleBase
-from tico.quantization.wrapq.wrappers.registry import register
+from tico.quantization.wrapq.wrappers.registry import try_register
 
 
-@register(nn.SiLU)
+@try_register("torch.nn.SiLU", "transformers.activations.SiLUActivation")
 class QuantSiLU(QuantModuleBase):
     """
-    QuantSiLU — drop-in replacement for nn.SiLU that quantizes
+    QuantSiLU — drop-in quantized implementation of the SiLU operation.
     both intermediate tensors:
```

Comment:
```suggestion
    It quantizes both intermediate tensors:
```
The following sentence becomes unnatural. How about refining it?